Enterprise-CMCS/macpro-mdct-qmr
# Tech Spec Changes - Changes to N/D/R Set(s)

### Description

Changes to NDR labels for Health Home measures for 2023.

### Related ticket(s)


MDCT-2384

---
### How to test

1) Sign into QMR using user: stateuserWA@test.com
2) Go to Reporting Year 2023
3) Review each measure below. To see the changes, select the first radio button for the first 3 questions and that will enable the N/D/R inputs

COL-HH
- New N/D/R Set: **Ages 46 to 49**. 
- Note: Typo in PDF, confirmed with David that the number is suppose to 46 and not 45 for the sentence too.

IEH-HH
-  Review PDF for text changes, pages 1 - 5 (MDCT-2384)

AIF-HH 
- was completed in MDCT-2260

IU-HH
- Removed maternity as a column 

### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
